src/Database/Types/Mysql/TimestampType.php

```diff
@@ -11,6 +11,8 @@ class TimestampType extends Type
 
     public function getSQLDeclaration(array $field, AbstractPlatform $platform)
     {
-        return 'timestamp';
+        $null = isset($field['default']) ? '' : ' null';
```

Comment:
It's case insensitive. But all types are using lower case. It's just a matter of consistency.